test/twoagent-monitored.py

```diff
@@ -0,0 +1,16 @@
+import os
+from autogen import AssistantAgent, UserProxyAgent, config_list_from_json
+from agentops import Client
+
+ao_client = Client(api_key=os.environ.get('AGENTOPS_API_KEY'),
```

Comment:
Secrets need to be updated in other relevant files. These include:
- https://github.com/microsoft/autogen/blob/9cec541630af22c2e2683d1f44a6ddd1d2e2425f/test/oai/test_utils.py#L17
- https://github.com/microsoft/autogen/blob/9cec541630af22c2e2683d1f44a6ddd1d2e2425f/OAI_CONFIG_LIST_sample

Reply:
I added a placeholder to the first file, `test_utils.py`.

`OAI_CONFIG_LIST` only has variables corresponding to OpenAI. All other environment variables seem to be expected to be exported previous to running scripts.